source/schemas/types.txt

```diff
@@ -509,3 +509,124 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. The 
+Realm SDKs support the creation of numerous types of geospatial data. However, 
+only persisted GeoJSON Point data can be queried. 
```

Comment:
To @jonatlee's point in another comment, perhaps we could clarify here that there is no built-in geospatial BSON type (which is what the top half of this page talks about).

- If you're not using sync then you can treat geospatial data as a regular object and can query it in MongoDB with regular [geospatial operators](https://www.mongodb.com/docs/manual/geospatial-queries/).
- If you're using sync then there's a smaller set of operators (i.e. only 1 right now) and some additional considerations.